File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdateAdjustPlantsRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsUpdateAdjustPlantsRequestItem(
    @JsonProperty("AdjustCount")
    val adjustCount: Int? = null,
    @JsonProperty("AdjustReason")
    val adjustReason: String? = null,
    @JsonProperty("AdjustmentDate")
    val adjustmentDate: String? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("Label")
    val label: String? = null,
    @JsonProperty("ReasonNote")
    val reasonNote: String? = null
)

File: sdks/thunkmetrc-go/wrapper/models/plants_update_adjust_plants_request_item.go
package models

type PlantsUpdateAdjustPlantsRequestItem struct {
    AdjustCount int `json:"AdjustCount,omitempty"`
    AdjustReason string `json:"AdjustReason,omitempty"`
    AdjustmentDate string `json:"AdjustmentDate,omitempty"`
    Id int `json:"Id,omitempty"`
    Label string `json:"Label,omitempty"`
    ReasonNote string `json:"ReasonNote,omitempty"`
}
